addon-scheduler/src/main/java/org/vaadin/stefan/fullcalendar/Resource.java

```diff
@@ -118,11 +117,11 @@ public Resource(String id, String title, String color, Collection<Resource> chil
      * @param children children (optional)
      * @param businessHours businessHours (optional)
      */
-    public Resource(String id, String title, String color, Collection<Resource> children, BusinessHours businessHours) {
+    public Resource(String id, String title, String color, Collection<Resource> children, BusinessHours... businessHours) {
```

Comment:
This will lead to issues with the previous constructor `public Resource(String id, String title, String color, Collection<Resource> children)`. That one can to be removed then or needs to have an updated parameter (with an empty array for instance).

Reply:
Removed unnecessary constructor